### PR TITLE
Update README.

### PR DESCRIPTION
**Pull request by rwjblue**

### CODE REVIEW

**Comment by rwjblue** [comment]
@GavinJoyce - Mind reviewing?

File: README.md [hunk 1]
```diff
@@ -12,24 +12,12 @@ the performance of the Ember.js framework. The general strategy is:
...
-ship with the performance suite, but you might encounter it if you are
-using a custom build.
-
 ### To run in development mode
 
 1. `npm install`
 2. `npm install -g bower`
 3. `bower install`
-4. `npm start`
+4. `npm run server`
```
Comment (GavinJoyce) [inline README.md]: shouldn't this be `ember server`?

Comment (GavinJoyce) [inline README.md]: sorry, I see you added the command below. please ignore

**Comment by GavinJoyce** [comment]
Looks good :+1:

**Comment by rwjblue** [comment]
Thanks for reviewing!

**Pull request merged**